src/gui/src/mainWindow.cpp

```diff
@@ -437,12 +448,11 @@ const std::string MainWindow::requestUserInput(const QString& title, const QStri
   return text.toStdString();
 }
 
-void MainWindow::setDb(odb::dbDatabase* db)
+void MainWindow::setDb()
```

Comment:
I think you can inline this since there seems to be just one caller and its only three lines.